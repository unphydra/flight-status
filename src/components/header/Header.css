.header-toolbar {
  margin: auto 20px;
  display: flex;
  flex-direction: row;
  justify-content: center;
}
