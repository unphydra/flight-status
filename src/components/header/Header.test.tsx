import { render, screen } from "@testing-library/react";
import { MemoryRouter as Router } from "react-router-dom";
import Header from "./index";

describe("Header", () => {
  it("Should renders without crashing", () => {
    const { container } = render(
      <Router>
        <Header />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });

  it("should render app bar title", () => {
    render(
      <Router>
        <Header />
      </Router>
    );
    const appBarTitleText = screen.getByText(/Flight Status/i);
    expect(appBarTitleText).toBeInTheDocument();
  });
});
