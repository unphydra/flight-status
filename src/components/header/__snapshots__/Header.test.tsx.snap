// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > Should renders without crashing 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1n8qf78-MuiToolbar-root"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary css-1e6y48t-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <div
          class="MuiTypography-root MuiTypography-h2 css-up0o9l-MuiTypography-root"
        >
          Flight Status
        </div>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
  </header>
</div>
`;
